plugins/modules/mysql_info.py

```diff
@@ -206,6 +206,19 @@
   type: dict
   sample:
   - { "2": { "Host": "", "Master_id": 1, "Port": 3306 } }
+connector_name:
+  description: Name of the python connector used by the plugin. When the driver is not identified, returns C(Unknown).
```

Comment:
```suggestion
  description: Name of the python connector used by the module. When the connector is not identified, returns C(Unknown).
```
i suggest using one of the term. If you prefer `driver` instead of `connector`, let's use it. If connector is fine, could you please change the function names to and other occurances?

Reply:
You mean refactor all the code to change "driver" to "connector" everywhere? So even `mysql_driver_fail_msg` -> `mysql_connector_fail_msg`? It's only 99 occurrences, but I want to be sure I understood correctly before.

Reply:
i think only the code you added would be fine. but you can use `driver` instead of `connector`, up to you. the goal is to make it consistent in terms of terminology. any of them is fine

Reply:
i would leave legacy code as is